test/unittest/meson.build

```diff
@@ -34,13 +34,10 @@ test_target = [
   'unittest_nntrainer_modelfile',
   'unittest_nntrainer_models',
   'unittest_nntrainer_graph',
-  'unittest_nntrainer_appcontext'
+  'unittest_nntrainer_appcontext',
+  'unittest_nntrainer_profiler'
 ]
 
-if get_option('enable-profile')
```

Comment:
any reason to remove this?

Reply:
eventhough enable-profile is off, unittest is runnable, without this the test coverage is 0% for the profiler

Reply:
then how about instead of removing this, enable profile in the CI build (with unit_test).

Reply:
Updated